test_scripts/API/Restructuring_OnResetTimeout/014_BC_OnResetTimeout_Alert_with_softButton.lua

```diff
@@ -0,0 +1,78 @@
+------------------------------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0189-Restructuring-OnResetTimeout.md
+------------------------------------------------------------------------------------------------------------------------
+-- Description: Check SDL does not reset timeout for Mobile app response for a specific RPC in case
+--  HMI sends 'OnResetTimeout(resetPeriod)' notification
```

Comment:
Can you please point to specifications for this?

Reply:
@atiwari9 Please see above